src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchCommand.cs

```diff
@@ -2,84 +2,113 @@
         {
-            app.Command("search", pkgSearch =>
+            var searchCommand = new CliCommand("search", Strings.pkgSearch_Description);
+
+            var searchTerm = new CliArgument<string>("Search Term")
+            {
+                Description = Strings.pkgSearch_termDescription,
+                Arity = ArgumentArity.ZeroOrOne,
+            };
+
+            var sources = new CliOption<List<string>>("--source")
+            {
+                Description = Strings.pkgSearch_SourceDescription,
+                Arity = ArgumentArity.ZeroOrMore
+            };
+
+            var exactMatch = new CliOption<bool>("--exact-match")
+            {
+                Description = Strings.pkgSearch_ExactMatchDescription,
+                Arity = ArgumentArity.Zero
+            };
+
+            var prerelease = new CliOption<bool>("--prerelease")
+            {
+                Description = Strings.pkgSearch_PrereleaseDescription,
+                Arity = ArgumentArity.Zero
+            };
+
+            var interactive = new CliOption<bool>("--interactive")
             {
-                pkgSearch.Description = Strings.pkgSearch_Description;
-                CommandOption help = pkgSearch.HelpOption(XPlatUtility.HelpOption);
-                CommandArgument searchTerm = pkgSearch.Argument(
-                    "<Search Term>",
-                    Strings.pkgSearch_termDescription);
-                CommandOption sources = pkgSearch.Option(
-                    "--source",
-                    Strings.pkgSearch_SourceDescription,
-                    CommandOptionType.MultipleValue);
-                CommandOption exactMatch = pkgSearch.Option(
-                    "--exact-match",
-                    Strings.pkgSearch_ExactMatchDescription,
-                    CommandOptionType.NoValue);
-                CommandOption prerelease = pkgSearch.Option(
-                    "--prerelease",
-                    Strings.pkgSearch_PrereleaseDescription,
-                    CommandOptionType.NoValue);
-                CommandOption interactive = pkgSearch.Option(
-                    "--interactive",
-                    Strings.pkgSearch_InteractiveDescription,
-                    CommandOptionType.NoValue);
-                CommandOption take = pkgSearch.Option(
-                    "--take",
-                    Strings.pkgSearch_TakeDescription,
-                    CommandOptionType.SingleValue);
-                CommandOption skip = pkgSearch.Option(
-                    "--skip",
-                    Strings.pkgSearch_SkipDescription,
-                    CommandOptionType.SingleValue);
-
-                pkgSearch.OnExecute(async () =>
+                Description = Strings.pkgSearch_InteractiveDescription,
+                Arity = ArgumentArity.Zero
+            };
+
+            var help = new HelpOption("--help", "-h", "-?")
+            {
+                Arity = ArgumentArity.Zero
+            };
```

Comment:
Is this needed? I thought System.CommandLine automatically adds the help option (at least `--help` and `-h`)

Reply:
fixed